packages/components/src/color-palette/index.tsx

```diff
@@ -214,10 +214,32 @@ const areColorsMultiplePalette = (
 	);
 };
 
+const getComputedBackgroundColorValueFromRef = (
+	value: string | undefined,
+	ref: RefObject< HTMLElement > | null
+) => {
+	let computedValue = value;
```

Comment:
Nit: The naming is kind of misleading to me. It only returns a "computed value from ref" when the value is a CSS var. Maybe something like `normalizeColorValue()` or `getValueForColord()`? And we can probably remove the `let computedValue` if we do an early return like:

```jsx
if ( ! currentValueIsCssVariable ) {
  return value;
}
```

Reply:
I have changed the function name.
Also, I think I could have simplified the function content without using `let` 👍

Reply:
Awesome, I like how these small changes improve readability 😊